scripts/badger/process_bribes_bvecvx.py

```diff
@@ -53,6 +53,7 @@ def step0_1(sim=False):
 
     for addr, mantissa in claimed.items():
         order_payload, order_uid = SAFE.badger.get_order_for_processor(
+            PROCESSOR,
             sell_token=SAFE.contract(addr),
             mantissa_sell=mantissa,
```

Comment:
```suggestion
            mantissa_sell=int(mantissa),
```
breaks when using `sim=True` otherwise

Reply:
i think it is casting [here](https://github.com/Badger-Finance/badger-multisig/blob/main/great_ape_safe/ape_api/cow.py#L53) later already within the cow class itself

Reply:
aaaaah that line needs to be `mantissa_sell = int(mantissa_sell)`! it is bugged using `==` currently.